runtime/src/assets/stylesheets/shared/_variables.less

```diff
@@ -1,4 +1,4 @@
-@import (reference) '../../../../node_modules/patternfly/src/less/variables';
+@import (reference) '../../../../node_modules/patternfly/dist/less/variables';
```

Comment:
Is there any specific reason to change this path?

Reply:
Distributables should be distributed from `dist`
It's crazy that it was using `src` in the first place, and somehow slipped through the code-reviews.

Reply:
Yes, and it was not working anymore with my setup. I think some step in the build stripped the src from the dependency.